src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -170,11 +171,12 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
     })
   }
 
-  submitOffer() {
+  submitOffer(setupIntentId) {
```

Comment:
this should have a type annotation. Do we always setupIntentId for an offer order now?